prisma/schema.prisma

```diff
@@ -0,0 +1,32 @@
+// This is your Prisma schema file,
+// learn more about it in the docs: https://pris.ly/d/prisma-schema
+
+datasource db {
+  provider = "postgresql"
+  url      = env("DATABASE_URL")
+}
+
+generator client {
+  provider      = "prisma-client-js"
+  binaryTargets = ["linux-musl", "native"]
+  output        = "../generated/prisma"
+}
+
+model Subscription {
+  id                         Int       @id @default(autoincrement())
+  account                    String    @unique
```

Comment:
@milenakrawczyk would the`unique` constraint prevent `alice.near` from enabling push notifications on two clients? e.g. her desktop browser and a mobile browser

Reply:
Yes, I made this simplification to show/test the full flow but might try to implement multiple subscriptions if this is preferred for the first stage.

Reply:
Yes, without multiple subscriptions per account, testing would be more difficult.